datalake/src/main/java/com/sequenceiq/datalake/service/sdx/StackRequestManifester.java

```diff
@@ -276,11 +276,17 @@ void setupCloudStorageAccountMapping(StackV4Request stackRequest, String environ
     }
 
     void validateMappingsConfig(MappingsConfig mappingsConfig, StackV4Request stackRequest) {
+        LOGGER.debug("Validating IDBMMS mapping for this cluster: {}", JsonUtil.writeValueAsStringSilent(mappingsConfig));
+        // Just to make it a bit more defensive
+        if (mappingsConfig == null || mappingsConfig.getActorMappings() == null || mappingsConfig.getActorMappings().isEmpty()) {
+            LOGGER.error("We have not found cloud storage access mapping for this cluster! {}", JsonUtil.writeValueAsStringSilent(mappingsConfig));
+            throw new BadRequestException("We have not found cloudstorage access mapping for this cluster!");
+        }
         // Validate RAZ if enabled, making sure that the RAZ mapping exists when it is required.
         if (stackRequest.getCluster().isRangerRazEnabled()) {
             if (!mappingsConfig.getActorMappings().containsKey("rangerraz")) {
-                LOGGER.error("IDBMMS mappings must contain the RAZ role if RAZ is to be created!");
-                throw new BadRequestException("IDBMMS mappings must contain the RAZ role if RAZ is to be created!");
+                LOGGER.error("Cloud storage access mapping must contain the RAZ role if RAZ is to be created!");
+                throw new BadRequestException("Cloud storage access mapping must contain the RAZ role if RAZ is to be created!");
```

Comment:
The associated CLI commands is called `set-id-broker-mappings`, and IDBroker is mentioned in CDP docs, so it could be safely mentioned here as well (perhaps as `Cloud storage access (IDBroker) mapping`). IDBMMS is definitely an internal detail, though.

Reply:
Changed to  Cloud storage access (IDBroker) mapping